push.js

```diff
@@ -0,0 +1,10 @@
+const { exec } = require('child_process');
+const version = require('./package.json').version;
+
+dir = exec(`gem push huginn_record_link_agent-${version}.gem`, (err, stdout, stderr) => {
+  if (err) {
+    console.log(err);
```

Comment:
Usually, you don't use `console.log` directly. Does this project use winston or something yet? You should at least be using the levels like `console.error` or whatever it is.